### PR TITLE
[doc] added a bit details to initial setup

### PR DESCRIPTION
**Pull request by min-xu-ai**
- [question] should we merge requirements-dev and requirements-test?
  I am not sure the differences.

testing: ran the testing commands.

# Before submitting

- [No ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ Yes] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ Yes] Did you make sure to update the docs?
- [ N/A] Did you write any new necessary tests?

## What does this PR do?
Fixes # (issue). N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃 - sort of.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,16 +31,41 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 disclosure of security bugs. In those cases, please go through the process
 outlined on that page and do not file a public issue.
 
+## Environment setup
+
+```
+~$ python3 -m venv venv2
+~$ source venv2/bin/activate
+(venv2) ~$ cd git/fairscale/
+(venv2) ~/git/fairscale (min/setup)$ pip3 install -r requirements-test.txt
```
Comment (msbaines) [inline CONTRIBUTING.md:40]: maybe remove the bash prompt from here, especially the min/setup part. It might be confusing.

Comment (min-xu-ai) [reply CONTRIBUTING.md:40]: good catch. will do.

**Comment by min-xu-ai** [comment]
I added a TODO on circleCI status/output? Is it needed?

Also, do we want to merge requirements-dev and requirement-test? Or document their differences?

**Comment by min-xu-ai** [comment]
Thank for approving! I will merge it after clean up some minor things.

**Pull request merged**